src/setupTests.ts

```diff
@@ -3,6 +3,9 @@
 // expect(element).toHaveTextContent(/react/i)
 // learn more: https://github.com/testing-library/jest-dom
 import '@testing-library/jest-dom/extend-expect';
+import { WebSocket } from 'mock-socket';
+
+window.WebSocket = WebSocket;
```

Comment:
This ensures that the client is using the mocked `WebSocket` as well, so the mocked server will receive connections.